client/dashboard/app/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Breadcrumbs as BreadcrumbsComponent } from '@automattic/components/src/breadcrumbs';
+import { useMatches } from '@tanstack/react-router';
+import type { BreadcrumbItemProps } from '@automattic/components/src/breadcrumbs/types';
+
+export default function Breadcrumbs() {
+	const matches = useMatches();
+
+	const items = [] as BreadcrumbItemProps[];
+
+	let hasParent = false;
+	matches.forEach( ( match ) => {
+		const breadcrumbItemLabel = match.staticData?.breadcrumbItemLabel;
+		if ( breadcrumbItemLabel || hasParent ) {
```

Comment:
I'd expect extracting the breadcrumbs based on how we register the routes (route.addChildren). Isn't that possible? It seems weird to have a `hasParent` flag based on `breadcrumbItemLabel` existence.